test.js

```diff
@@ -56,3 +56,39 @@ test('friendly names', function(t) {
 	t.equal(map.delete, map.unset)
 	t.end()
 })
+
+test('no constructor', function (t) {
+	var map = new KeyMaster()
+	
+	map.set('key1', 3)
+	map.set('key1')
+	map.set('key2')
+	
+	t.equal(map.get('key1'), undefined)
+	t.equal(map.get('key2'), undefined)
+	t.end()
+})
+
+test('set and delete return', function (t) {
+	var map = new KeyMaster()
+	
+	t.equal(map.set('key1', 3), 3)
+	t.equal(map.set('key2'), undefined)
+	t.ok(map.remove('key1'))
+	t.notOk(map.remove('key1'))
```

Comment:
Delete always returns true unless the property is configurable.  If we want `remove` to return based on whether or not the property existed, we'll have to change the code.